zipkin-lens/src/components/App/Layout.jsx

```diff
@@ -13,33 +13,90 @@
  */
 import PropTypes from 'prop-types';
 import React from 'react';
+import { faSearch, faProjectDiagram, faHome } from '@fortawesome/free-solid-svg-icons';
+import { faGithub, faTwitter, faGitter } from '@fortawesome/free-brands-svg-icons';
 import Box from '@material-ui/core/Box';
+import Drawer from '@material-ui/core/Drawer';
 import CssBaseline from '@material-ui/core/CssBaseline';
+import List from '@material-ui/core/List';
+import { makeStyles } from '@material-ui/styles';
 
-import Sidebar from './Sidebar';
+import SidebarMenu from './SidebarMenu';
+import Logo from '../../img/zipkin-logo.svg';
+
+const drawerWidth = '3.2rem';
+
+const useStyles = makeStyles(theme => ({
```

Comment:
This style is almost the same as the previous one.

Reply:
This combines the styles of the previous Sidebar component and the previous Layout component.